metis-core/metis-core-service/src/main/java/eu/europeana/metis/core/service/ProxiesService.java

```diff
@@ -394,9 +400,64 @@ public RecordsResponse getListOfFileContentsFromPluginExecution(MetisUserView me
     // Done.
     return new RecordsResponse(records);
   }
+  /**
+   * Get a record from the external resource based on o searchId, workflow execution and {@link PluginType}.
+   *
+   * @param metisUserView the user wishing to perform this operation
+   * @param workflowExecutionId the execution identifier of the workflow
+   * @param pluginType the {@link ExecutablePluginType} that is to be located inside the workflow
+   * @param searchId the searchId of the record we wish to obtain
```

Comment:
What is the `searchId`? It is not clear to me what this is supposed to be, the `recordId`? `europeanaId`? `anyOtherId`?
If we are *not* supposed to know then perhaps change the name to `idToSearchWith`?

Reply:
OK, changed it to `idToSearch`. Is that better?